include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -13400,6 +13400,33 @@
           "extensions" : [ "SPV_INTEL_fpga_memory_attributes" ],
           "version" : "None"
         },
+        {
+          "enumerant" : "StridesizeINTEL",
+          "value" : 5883,
+          "parameters" : [
+            { "kind" : "LiteralInteger", "name" : "'Stride Size'" }
+          ],
+          "capabilities" : [ "FPGAMemoryAttributesINTEL" ],
+          "extensions" : [ "SPV_INTEL_fpga_memory_attributes" ],
```

Comment:
Extension line is now considered redundant as this information present in the capabilities section as well. While we are here, may I ask you to remove it for decorations under `FPGAMemoryAttributesINTEL` capability?

Reply:
Done